go.mod

```diff
@@ -0,0 +1,48 @@
+module numaflow-nats-source
```

Comment:
should this now be `module github.com/numaproj/numaflow-contrib/numaflow-nats-source`?